src/main/java/teammates/common/util/LogHelper.java

```diff
@@ -0,0 +1,192 @@
+package teammates.common.util;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import teammates.common.exception.InvalidParametersException;
+
+import com.google.appengine.api.log.AppLogLine;
+import com.google.appengine.api.log.LogQuery;
+import com.google.appengine.api.log.LogServiceFactory;
+import com.google.appengine.api.log.RequestLogs;
+import com.google.appengine.api.log.LogService.LogLevel;
+
+public class LogHelper {
+    /**
+     * 6 versions to query, including the current version and its 5 preceding versions.
+     */
+    private static final int MAX_VERSIONS_TO_QUERY = 6;
```

Comment:
This comment should be a description of the variable rather than the value. But you can value as an example to explain the variable.